README.rst

```diff
@@ -216,12 +203,16 @@ our model development environment, alongside the seed VM.
 
 .. code-block:: console
 
-   # NOTE: Make sure to use ./tenks, since just ‘tenks’ will install via PyPI.
-   export TENKS_CONFIG_PATH=config/src/kayobe-config/tenks.yml
-   ./dev/tenks-deploy-overcloud.sh ./tenks
+   # Set Environment variables for Kayobe dev scripts
+   export KAYOBE_CONFIG_SOURCE_PATH=~/deployment/src/kayobe-config
+   export KAYOBE_VENV_PATH=~/deployment/venvs/kayobe
```

Comment:
Is this necessary given you ran `source ~/deployment/env-vars.sh` earlier?

Reply:
Unfortunately, those are magic vars that are only used in the kayobe dev scripts, so they're not exported by beokay's `env-vars.sh`

Reply:
Thanks, I thought they might be part of the `kayobe-env` script, but you're right they aren't.